lib/http.js

```diff
@@ -96,18 +96,8 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       }
 
       const {controller, referenceId} = req.query;
-      if(!controller) {
-        throw new BedrockError(
-          'Query not supported; a "controller" must be specified.',
-          'NotSupportedError', {public: true, httpStatusCode: 400});
-      }
-      if(!referenceId) {
-        // query for all keystores controlled by controller not implemented yet
-        // TODO: implement
-        throw new BedrockError(
-          'Query not supported; a "referenceId" must be specified.',
-          'NotSupportedError', {public: true, httpStatusCode: 400});
-      }
+      // query for all keystores controlled by controller not implemented yet
+      // TODO: implement
```

Comment:
is this TODO necessary?  Could it just be a github issue for an enhancement. I'm trying to figure out how NOT to have TODOs and FIXMES piling up hidden away in code that involves another massive research project.

Reply:
Removed and added as an issue. Also did the same for another TODO a few lines below.

// TODO: implement `update` for keystore config